### PR TITLE
Add project and community section in docs

### PR DESCRIPTION
**Pull request by arturo-seijas**
Applicable spec: N/A
### Overview

Add project and community section in docs

### Rationale

N/A

### Juju Events Changes

N/A

### Module Changes

N/A

### Library Changes

N/A

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by varshigupta12 — DISMISSED** [review]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -4,15 +4,27 @@ The code is licensed under the [Apache License, version 2](https://www.apache.or
...
+## Project and community
+
+The SAML Integrator Operator is a member of the Ubuntu family. It's an open source
+project that warmly welcomes community projects, contributions, suggestions,
+fixes and constructive feedback.
+* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+* [Get support](https://discourse.charmhub.io/)
+* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
+* [Contribute](https://charmhub.io/indico/docs/how-to-contribute)
+Thinking about using the Indico Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
```
Comment (cbartz) [inline docs/index.md:29]:
```suggestion
Thinking about using the SAML Integrator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
```

File: docs/index.md [hunk 2]
```diff
@@ -4,15 +4,27 @@ The code is licensed under the [Apache License, version 2](https://www.apache.or
...
-## In this documentation
+## Project and community
+
+The SAML Integrator Operator is a member of the Ubuntu family. It's an open source
+project that warmly welcomes community projects, contributions, suggestions,
+fixes and constructive feedback.
+* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+* [Get support](https://discourse.charmhub.io/)
+* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
+* [Contribute](https://charmhub.io/indico/docs/how-to-contribute)
```
Comment (cbartz) [inline docs/index.md:28]:
```suggestion
* [Contribute](https://charmhub.io/saml-integrator/docs/how-to-contribute)
```

File: docs/index.md (hunk 1 above)
Comment (arturo-seijas) [reply docs/index.md:29]: Thx!

**Review by cbartz — DISMISSED** [review]
LGTM

**Review by mthaddon — APPROVED** [review]

**Review by cbartz — APPROVED** [review]

**Pull request merged**